.gitignore

```diff
@@ -20,6 +20,7 @@ Thumbs.db
 .tmproj
 *.sublime-project
 *.sublime-workspace
+*.lnk
```

Comment:
It should really be `git rebase -i master`, as master is the origin of your fix177 branch.
Then yes, after editing and checking the result, `git push --force origin fix177`.